ape-config.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
     version: 4.8.2
   - name: tokenized-strategy
     github: yearn/tokenized-strategy
-    branch: 3.0.0
+    branch: v3.0.0
```

Comment:
I think it should be tag v3.0.0 if that is possible in ape config. Or `version` as for open zeppelin above

Reply:
Well there actuall is both a branch and tag for v3.0.0 to make sure it works as much as possible.

But ape's newest version normally wants to use "ref" so updated to that